### ISSUE TITLE
NameError: undefined local variable or method `path_separator'

### ISSUE THREAD

**Issue by ponnusamycto** [opscode/knife-windows#154]
I am running knife bootstrap command from my workstation( which is windows) and my target bootstrap system also windows. while executing below command getting below error message. Please give solution for this.

```
C:\chef>knife bootstrap windows winrm Desktop1.test.com -x user1 -P password1
Bootstrapping Chef on Desktop1.test.com
ERROR: knife encountered an unexpected error
This may be a bug in the 'bootstrap windows ssh' knife command or plugin
Please collect the output of this command with the `-VV` option before filing a
bug report.
Exception: NameError: undefined local variable or method `path_separator' for #<
Chef::Knife::Core::WindowsBootstrapContext:0x31ff3b0>


C:\chef>knife bootstrap windows ssh Desktop1.test.com
Bootstrapping Chef on Desktop1.test.com
ERROR: knife encountered an unexpected error
This may be a bug in the 'bootstrap windows ssh' knife command or plugin
Please collect the output of this command with the `-VV` option before filing a
bug report.
Exception: NameError: undefined local variable or method `path_separator' for #<
Chef::Knife::Core::WindowsBootstrapContext:0x31ff3b0>


C:\chef>knife bootstrap windows ssh Desktop1.test.com -VV
Bootstrapping Chef on Desktop1.test.com
DEBUG: Looking for bootstrap template in E:/opscode/chef/embedded/lib/ruby/gems/
1.9.1/gems/knife-windows-1.0.0.dev.0/lib/chef/knife/bootstrap
DEBUG: Found bootstrap template in E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/
gems/knife-windows-1.0.0.dev.0/lib/chef/knife/bootstrap
E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0.0.dev.0/lib/
chef/knife/core/windows_bootstrap_context.rb:267:in `cleanpath': undefined local
 variable or method `path_separator' for #<Chef::Knife::Core::WindowsBootstrapCo
ntext:0x2fc89c8> (NameError)
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/core/windows_bootstrap_context.rb:249:in `escape_glob'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/core/windows_bootstrap_context.rb:238:in `trusted_certs_
content'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/core/windows_bootstrap_context.rb:56:in `trusted_certs_s
cript'
        from (erubis):200:in `block in evaluate'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/erubis-2.7.0/lib/
erubis/evaluator.rb:74:in `instance_eval'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/erubis-2.7.0/lib/
erubis/evaluator.rb:74:in `evaluate'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/bootstrap_windows_base.rb:138:in `render_template'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/bootstrap_windows_base.rb:183:in `create_bootstrap_bat_c
ommand'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/bootstrap_windows_base.rb:154:in `bootstrap'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/knife-windows-1.0
.0.dev.0/lib/chef/knife/bootstrap_windows_ssh.rb:75:in `run'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/chef-11.16.4-x86-
mingw32/lib/chef/knife.rb:493:in `run_with_pretty_exceptions'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/chef-11.16.4-x86-
mingw32/lib/chef/knife.rb:174:in `run'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/chef-11.16.4-x86-
mingw32/lib/chef/application/knife.rb:139:in `run'
        from E:/opscode/chef/embedded/lib/ruby/gems/1.9.1/gems/chef-11.16.4-x86-
mingw32/bin/knife:25:in `<top (required)>'
        from E:/opscode/chef/bin/knife:23:in `load'
        from E:/opscode/chef/bin/knife:23:in `<main>'
```

My gem env details are below 

```
RubyGems Environment:
  - RUBYGEMS VERSION: 1.8.28
  - RUBY VERSION: 1.9.3 (2013-11-22 patchlevel 484) [i386-mingw32]
  - INSTALLATION DIRECTORY: E:/opscode/chef/embedded/lib/ruby/gems/1.9.1
  - RUBY EXECUTABLE: E:/opscode/chef/embedded/bin/ruby.exe
  - EXECUTABLE DIRECTORY: E:/opscode/chef/embedded/bin
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86-mingw32
  - GEM PATHS:
     - E:/opscode/chef/embedded/lib/ruby/gems/1.9.1
     - C:/Users/ponnusamy/.gem/ruby/1.9.1
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :benchmark => false
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - http://rubygems.org/
```

**Comment by technovationcenter** [comment]
It's woking with older version.. Thanks for your help/

**Comment by btm** [comment]
Great, this should get fixed in master by #151, #152, #157.
